src/engraving/layout/v0/tlayout.cpp

```diff
@@ -3546,11 +3546,12 @@ void TLayout::layout(Note* item, LayoutContext&)
             if (item->negativeFretUsed()) {
                 item->setFretString(u"-" + item->fretString());
             }
-
+            // Don't use regular less than or more than symbols
+            // they are not centered. Use these instead! ＜＞
             if (item->displayFret() == Note::DisplayFretOption::ArtificialHarmonic) {
-                item->setFretString(String(u"%1 <%2>").arg(item->fretString(), String::number(item->harmonicFret())));
+                item->setFretString(String(u"%1 ＜%2＞").arg(item->fretString(), String::number(item->harmonicFret())));
```

Comment:
You should add a new vtest/modify an existing one

Reply:
There is no need to change anything in vtest, it is compared with the pictures taken from the reference build.
You just need to see if the expected changes have happened and mark the PR with the "vtest" label - means that vtest changes are expected.